symbolic-symcache/src/writer.rs

```diff
@@ -186,10 +193,62 @@ impl<'a> SymCacheConverter<'a> {
             fun_idx as u32
         };
 
-        let mut parent_line_records = parent_line_records.iter().peekable();
-        let mut parent_line = parent_line_records.next();
+        // We can divide the instructions in a function into two buckets:
+        //  (1) Instructions which are part of an inlined function call, and
+        //  (2) instructions which are *not* part of an inlined function call.
```

Comment:
the "part of" might be a bit confusing here.
maybe:

(1) Instructions which are in the middle of an inlined call chain, and
(2) instructions which do not have further callees.

That’s not too good of an explanation either though :-D